htdocs/web_portal/components/Draw_Components/draw_page_components.php

```diff
@@ -18,162 +18,184 @@
      * See the License for the specific language governing permissions and
      * limitations under the License.
      *
-     *====================================================== */
+     * PHPMD does not parse use of $properties with :: scope resolution
+     * @SuppressWarnings(PHPMD.UndefinedVariable)
+     */
 
     /* Draws the first part of the page (headers, left hand menu) */
```

Comment:
Why not unindent the comments as well?

Reply:
No reason. The auto-reformat didn't touch them, but I'll update.